packages/babel-helpers/src/helpers-generated.ts

```diff
@@ -13,7 +13,7 @@ function helper(minVersion: string, source: string) {
 }
 
 export default Object.freeze({
-  // size: 1126, gzip size: 512
+  // size: 1126, gzip size: 511
```

Comment:
I suspect the node.js in your environment is installed by homebrew, which links to the latest zlib version 1.3.1. Can you post the result of `node -e "console.log(process.versions.zlib)"`? 

For reference, it prints `1.2.13.1-motley-5daffc7` for the official Node.js 20 build.

Reply:
When I type node -e "console.log(process.versions.zlib)", it says 1.2.13.1-motley.